externals/sha3/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(SHA3)
+
+add_library(cvmfs_sha3 STATIC
+  src/64opt/KeccakF-1600-opt64.c
+  src/KeccakHash.c
+  src/KeccakSponge.c
+  src/SnP-FBWL-default.c
+  )
+
+set_property(TARGET cvmfs_sha3 PROPERTY POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
In the current build process, we take the upstream Makefile and add the cvmfs compiler flags.  Do we here compile with equivalent options?

Reply:
We should, as they are built as part of the cvmfs project itself, but it's something I can check to be sure if you want.